### DISCUSSION TITLE
[@types/zipcodes] wrong type of lookupByCoords

### DISCUSSION THREAD

**Discussion by bitcookr** [DefinitelyTyped/DefinitelyTyped#65709] [Issues with a @types package] [answered]
The current lookupByCoords returns ``string  |  null``.
It should be ``ZipCode | null``

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/128102810/4db73814-9b15-4870-a741-200b1f1c7713)

``export function lookupByCoords(lat: number, lon: number): ZipCode | null;``

[a comment by an automated account is not shown]

**Comment by bitcookr** [comment, marked as answer]
I made a PR for this solution.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65713

**Reply by peterblazejewicz** [reply]
@bitcooker thanks!
